src/components/widgets/accounts-cash-projection/accounts-cash-projection.directive.coffee

```diff
@@ -71,10 +74,12 @@ module.controller('WidgetAccountsCashProjectionCtrl', ($scope, $q, $filter, Impa
         legend:
           useHTML: true
           labelFormatter: ->
-            name = this.name
-            imgSrc = ImpacAssets.get(_.camelCase(name + 'LegendIcon'))
+            filename = this.name
```

Comment:
Please show the legend only for the main series: "Payables" and "Receivables - no need to have x6 legend entries as the tooltip should differentiate the different stacks.